Makefile

```diff
@@ -1,16 +1,41 @@
-.PHONY: all test clean
+.PHONY: build test test-memory test-mariadb test-postgresql test-http
 
 build:
 	opam exec -- dune build
 
 clean:
 	opam exec -- dune clean
 
-test:
+test-unit: build
 	SIHL_ENV=test opam exec -- dune test
 
-test-slow:
-	SIHL_ENV=test opam exec -- dune build @runtest-all
+test-memory: build
+	SIHL_ENV=test ./_build/default/sihl-queue/test/queue_memory.exe
+
+test-mariadb: build
+	SIHL_ENV=test ./_build/default/sihl-email/test/email_mariadb.exe
+	SIHL_ENV=test ./_build/default/sihl-persistence/test/database_mariadb.exe
+	SIHL_ENV=test ./_build/default/sihl-queue/test/queue_mariadb.exe
+	SIHL_ENV=test ./_build/default/sihl-storage/test/storage_mariadb.exe
+	SIHL_ENV=test ./_build/default/sihl-user/test/password_reset_mariadb.exe
+	SIHL_ENV=test ./_build/default/sihl-user/test/token_mariadb.exe
+	SIHL_ENV=test ./_build/default/sihl-user/test/user_mariadb.exe
+	SIHL_ENV=test ./_build/default/sihl-user/test/authn_mariadb.exe
+	SIHL_ENV=test ./_build/default/sihl-web/test/csrf_mariadb.exe
+	SIHL_ENV=test ./_build/default/sihl-web/test/session_mariadb.exe
+	SIHL_ENV=test ./_build/default/sihl-web/test/flash_mariadb.exe
+
+test-postgresql: build
```

Comment:
Why are there fewer tests for postgres?

Reply:
Because for now we are not using the postgres backends, for some services I have implemented just the backends we currently use.

Reply:
Ok, makes sense, thanks!